config-scripts/cups-directories.m4

```diff
@@ -266,6 +266,8 @@ AC_ARG_WITH([rundir], AS_HELP_STRING([--with-rundir], [set transient run-time st
     AS_CASE(["$host_os_name"], [darwin*], [
 	# Darwin (macOS)
 	CUPS_STATEDIR="$CUPS_SERVERROOT"
+    ], [solaris*], [
+	CUPS_STATEDIR="/system/volatile/cups"
```

Comment:
Not having used a recent version of Solaris, this looks like something that needs more checking - certainly the last version of Solaris that CUPS actually supported did not have a /system hierarchy, so I'd expect there to be a directory existence check.

Reply:
True, will update this.